README.md

```diff
@@ -1,19 +1,49 @@
 # hfst-optimized-lookup
 
+[![PyPI version](https://img.shields.io/pypi/v/hfst-optimized-lookup)](https://pypi.org/project/hfst-optimized-lookup/)
+
 A pip-installable library version of [hfst-optimized-lookup][], originally
 built for [itwêwina][].
 
-    $ pip install hfst-optimized-lookup
+## Install
+
+    pip install hfst-optimized-lookup
+
```

Comment:
```suggestion
This requires that the machine running `pip` have a working C++ compiler. If
enough people ask for us to upload [binary ‘wheels’] so that you don’t need a
compiler at install time, we could start doing so.

[binary ‘wheels’]: https://packaging.python.org/guides/distributing-packages-using-setuptools/#wheels
```